src/components/Table/Table.tsx

```diff
@@ -617,6 +655,32 @@ const Table = ({
         )}
       </thead>
       {createRows()}
+      {showFooter && (
+        <tfoot>
+          {width > minWidthBreakpoint && (
+            <StyledFooter
+              columnGap={columnGap}
+              columnWidths={columnWidths}
+            >
+              {Object.keys(copiedColumns).map((headerColumnKey: string, index: number) => {
+                const RenderedFooterCell = StyledFooterCell;
+                const breakpointHit =
+                  width > (copiedColumns[headerColumnKey].minTableWidth || Infinity);
+                // columns.map return
+                return (
+                  (!copiedColumns[headerColumnKey].minTableWidth || breakpointHit) && (
+                    <RenderedFooterCell key={`f${index}`}>
+                      {console.log(`f${index}`)
+                        // FOOTER CONTENT GOES HERE
```

Comment:
Seems like we should add footer content here

something like 
```suggestion
                        {column.footer}
```